src/TriangleColorPicker.js

```diff
@@ -264,7 +264,8 @@ export class TriangleColorPicker extends React.PureComponent {
           </View>
           }
         </View>
-        <View style={[styles.colorPreviews, computed.colorPreviews]}>
+        { this.props.hideOldColor === false && 
```

Comment:
and here... I believe the property won't hide old color but "both" colors - maybe wrong name? `hideColorPreview`?